chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchDetails.java

```diff
@@ -169,6 +169,12 @@
      */
     @XmlElement(required = false, nillable = true)
     private Integer countCqms;
+    
+    /**
+     * If this product has QMS.
+     */
+    @XmlElement(required = false, nillable = true)
+    private Boolean hasQms;
```

Comment:
I think that this either should not be in the domain object or should be marked as Transient. As it is written, this field would get included in all listings written out to XML files for download by the users but I think the field is more for our internal business logic and not something that should be a property on listings that users see.